README.md

```diff
@@ -146,6 +146,9 @@ Here are the options you can set in your `.cz-config.js`:
   }
   ```
 * **allowCustomScopes**: {boolean, default false}: adds the option `custom` to scope selection so you can still type a scope if you need.
+* **allowEmptyScopes**: {boolean, default false}: allow non-selection or not to fill in the scopes.
```

Comment:
I think `allowEmptyScopes` default should be true because it makes it easier for people to use this tool.  What are your thoughts?

Reply:
I agree

Reply:
If we make `allowEmptyScopes` default `true`, would that be a breaking change? No problems if so, we just need to make the commit text have:

 ```
BREAKING CHANGES: 
- describe the breaking change
- steps to upgrade
````